isolate_example/lib/main.dart

```diff
@@ -14,22 +14,63 @@
 
 import 'package:flutter/material.dart';
 
-void main() => runApp(MyApp());
+void main() {
+  runApp(startApp());
+}
 
-class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
+class startApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
-      home: Scaffold(
-        appBar: AppBar(
-          title: Text('Isolates'),
+      home: DefaultTabController(
+        length: 3,
+        child: Scaffold(
+          appBar: AppBar(
+            bottom: TabBar(
+              tabs: [
+                Tab(icon: Icon(Icons.flash_on), text: 'Performance'),
+                Tab(icon: Icon(Icons.sync), text: 'Infinite Process'),
+                Tab(icon: Icon(Icons.storage), text: 'Data Transfer'),
+              ],
+            ),
+            title: Text('Isolate Example'),
+          ),
+          body: TabBarView(
+            children: [
+              performacePage(),
+              infiniteProcessPage(),
+              dataTransferPage(),
+            ],
+          ),
         ),
-        body: Center(
-          child: Text('Here is my app'),
-        )
-      )
+      ),
     );
   }
-}
\ No newline at end of file
+}
+
+class performacePage extends StatelessWidget {
```

Comment:
PerformancePage.

Dart uses UpperCamelCase (I call it "title case," personally) for type names:

https://dart.dev/guides/language/effective-dart/style#do-name-types-using-uppercamelcase